src/Soil/Biogeochemistry/Biogeochemistry.jl

```diff
@@ -137,6 +158,311 @@ function DETECTModelParameters(;
     )
 end
 
-include("./co2_parameterizations.jl")
+
+
+
+#################################
+
+"""
+    DETECTModel
```

Comment:
Is DETECT the best name for this?

Reply:
This is a good question, I feel like if we start using the DETECT model as is, we really need to acknowledge it somehow. But if we modify it, then we can name if differently? (but still acknowledge the inspiration). 

The question is more about consistency with other models in ClimaLSM - is Richard equation called RichardModel? Or something more concrete like SoilMoistureModel? 

We should just do the same everywhere

Reply:
I replaced all `DETECT` occurrences with `soilco2` for now, we can always change later.